cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -75,7 +75,8 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["sleep", "30"]
+              sleep:
```

Comment:
The `exec` should be replaced by `sleep`

Thanks for following up on this!

Reply:
Late night PR :facepalm: 
Fixed!